frontend/src/metabase/ui/components/overlays/Menu/Menu.styled.tsx

```diff
@@ -13,10 +13,11 @@ export const getMenuOverrides = (): MantineThemeOverride["components"] => ({
         padding: "0.75rem !important",
         minWidth: "11.5rem",
         overflow: "auto",
-        background: theme.fn.themeColor("bg-white"),
+        background: "var(--mb-color-background)",
+        borderColor: "var(--mb-color-border)",
```

Comment:
We need border for dark theme, since without it, we couldn't really see the popover